server/collectionAttribution/queries.ts

```diff
@@ -1,28 +1,34 @@
 import ensureUserForAttribution from 'utils/ensureUserForAttribution';
 import { CollectionAttribution, includeUserModel } from 'server/models';
+import { expect } from 'utils/assert';
 
-export const createCollectionAttribution = (inputValues) => {
-	return CollectionAttribution.create({
+export const createCollectionAttribution = async (inputValues: {
+	userId?: string | null;
+	collectionId: string;
+	name: string;
+	order: number;
+	communityId?: string | null;
+}) => {
+	const newAttribution = await CollectionAttribution.create({
```

Comment:
This is another example of being a bit too enthusiastic: turning `then`d functions into async functions. The reason is almost always that it is easier to `expect` things like this, like below.